beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -473,6 +475,28 @@ func defaultVoluntaryExitTopicParams() *pubsub.TopicScoreParams {
 	}
 }
 
+func defaultBlsToExecutionChangeTopicParams() *pubsub.TopicScoreParams {
+	return &pubsub.TopicScoreParams{
+		TopicWeight:                     voluntaryExitWeight,
```

Comment:
If this weight is really going to be a constant then we need to return the same as above and avoid code duplication.

Reply:
good catch, this is the incorrect weight. I will have to set a new one for our scoring params